app/components/e-threejs-demo/demos/cubes.hbs

```diff
@@ -0,0 +1,30 @@
+<Controls @onUpdate={{this.onCountUpdate}} />
+
+<main>
+  <EThreejs @id={{this.sceneId}} as |s|>
```

Comment:
Thoughts on:
```hbs
<EThree::Scene as |s|>
    <s.Lights.DirectionalLight @position={{this.lightPosition}} />
    <s.Cameras.PerspectiveCamera
      @position={{this.cameraPosition}}
      @aspectRatio={{this.aspectRatio}}
    />

    <s.Object3d @rotation={{this.containerRotation}} as |container|>

      {{#each this.appState.objectProperties as |properties|}}
        <container.Objects.Mesh
          @rotation={{properties.r}}
          @position={{properties.p}}
          @scale={{properties.s}}
          @material={{this.material}}
          @geometry={{this.geometry}}
        />
      {{/each}}

    </s.Object3d>

  </EThree::Scene>
</main>
```
?

Reply:
think what bothers me with approach is it feels like two API's: one for the template and another for what yields what. 

Overall for now I would somewhat prefer specifying the `@parent` to yielding complex structures from the scene / objects.